packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/eslint.config.mjs

```diff
@@ -1,7 +1,6 @@
 import azsdkEslint from "@azure/eslint-plugin-azure-sdk";
 
-export default [
-  ...azsdkEslint.configs.recommended,
+export default azsdkEslint.config([
```

Comment:
Do we need to update the HLC also?

Reply:
yes for azure js repo

Reply:
HLC does not include `eslint.config.mjs`